Fix the confusing message when detecting a Python interpreter under `python.use_venv=False`
